package.json

```diff
@@ -116,12 +113,11 @@
     "rollup-plugin-preserve-shebangs": "^0.2.0",
     "rollup-plugin-styles": "^4.0.0",
     "sloc": "^0.2.1",
-    "storybook": "^7.6.17",
+    "storybook": "^8.0.10",
     "style-loader": "^3.3.3",
     "typescript": "^5.4.2",
     "typescript-coverage-report": "^0.7.0",
-    "vite-plugin-istanbul": "^5.0.0",
-    "vite-plugin-turbosnap": "^1.0.3",
```

Comment:
turbosnap is now built into the `"@storybook/react-vite"` bundle and can be enabled with the standard `--stats-json` argument (see line 146 of this file).